README.es-ES.md

```
@@ -0,0 +1,59 @@
+[Pretraining Large version](https://huggingface.co/0x7194633/keyt5-large)
+|
+[Pretraining Base version](https://huggingface.co/0x7194633/keyt5-base)
+
+[habr article](https://habr.com/ru/post/599715/)
+
+# Uso
+Ejemplo de uso (el código devuelve una lista con palabras clave; es posible que haya duplicados):
+
+[![Try Model Training In Colab!](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/0x7o/notebooks/blob/main/T5/keyT5_use.ipynb)
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)